packages/api/test/__fixtures__/sdk/readme/index.ts

```diff
@@ -78,113 +79,140 @@ class SDK {
    */
   get(
     path: '/api-registry/{uuid}',
-    metadata: GetAPIRegistryMetadataParam
-  ): Promise<GetAPIRegistry_Response_200 | Error_REGISTRY_NOTFOUND>;
```

Comment:
`Error_REGISTRY_NOTFOUND` → `ErrorRegistryNotfound` is another example of us forcing type names to be pascal case.